hpx/runtime/actions/plain_action.hpp

```diff
@@ -72,90 +91,18 @@ namespace hpx { namespace actions
     /// \endcond
 }}
 
-/// \def HPX_REGISTER_PLAIN_ACTION(action_type)
-/// \brief Registers an existing free function as a plain action with HPX
-///
-/// The macro \a HPX_REGISTER_PLAIN_ACTION can be used to register an existing
-/// free function as a plain action. It additionally defines an action type
-/// named \a action_type.
-///
-/// The parameter \p action_type is the name of the action type to register
-/// with HPX.
-///
-/// \par Example:
-///
-/// \code
-///       namespace app
-///       {
-///           void some_global_function(double d)
-///           {
-///               cout << d;
-///           }
-///
-///           // This will define the action type 'app::some_global_action' which
-///           // represents the function 'app::some_global_function'.
-///           HPX_DEFINE_PLAIN_ACTION(some_global_function, some_global_action);
-///       }
-///
-///       // The following macro expands to a series of definitions of global objects
-///       // which are needed for proper serialization and initialization support
-///       // enabling the remote invocation of the function `app::some_global_function`.
-///       //
-///       // The second argument used has to be the same as used for the
-///       // HPX_DEFINE_PLAIN_ACTION above.
-///       HPX_REGISTER_PLAIN_ACTION(app::some_global_action, some_global_action);
-/// \endcode
-///
-/// \note Usually this macro will not be used in user code unless the intend is
-/// to avoid defining the action_type in global namespace. Normally, the use of
-/// the macro \a HPX_PLAIN_ACTION is recommend.
-///
-#define HPX_REGISTER_PLAIN_ACTION(...)                                        \
-    HPX_REGISTER_PLAIN_ACTION_(__VA_ARGS__)                                   \
-/**/
+namespace hpx { namespace traits {
```

Comment:
The comment above was used to generate real documentation. Has it been moved somewhere else?